ocaml/lang/security/exec.ml

```diff
@@ -0,0 +1,11 @@
+#load "unix.cma";;
+(* ruleid:ocamllint-exec *)
+let a = Unix.execve "ls"
+(* ruleid:ocamllint-exec *)
+let b = Unix.execvp "ls"
+(* ruleid:ocamllint-exec *)
+let c = Unix.execvpe "ls"
+(* ruleid:ocamllint-exec *)
+let d = Unix.system "ls"
+(* ruleid:ocamllint-exec *)
+let e = Sys.command "ls"
```

Comment:
I don't think any of these test cases are actual examples of possible command injection since they all have hardcoded commands and no interaction at all with potential user input.

Reply:
```suggestion
#load "unix.cma";;
let p = String.concat "ls" [" "; Sys.argv.(1)]
(* ruleid:ocamllint-exec *)
let a = Unix.execve p
(* ruleid:ocamllint-exec *)
let b = Unix.execvp p
(* ruleid:ocamllint-exec *)
let c = Unix.execvpe p
(* ruleid:ocamllint-exec *)
let d = Unix.system p
(* ruleid:ocamllint-exec *)
let e = Sys.command p
```